internal/exec/exec.go

```diff
@@ -303,6 +303,9 @@ func (r *request) addError(err *errors.QueryError) {
 func (r *request) handlePanic() {
 	if err := recover(); err != nil {
 		execErr := errors.Errorf("graphql: panic occured: %v", err)
+		if err, ok := err.(error); ok {
+			execErr.ResolverError = err
```

Comment:
Hmm, is there a good scenario on why we should set the field on a panic? A panic should always be an unexpected internal error, so there should be nothing to tell the user except "internal error".

Right now the error message includes the panic's error value, but eventually I want to change it to only say "internal error" and not leak internals. However right now I don't want to do that yet because there are still some graphql-go error conditions that cause a panic and they would be hard to understand if the message was omitted.